ui/i18n/src/main/res/values/strings.xml

```diff
@@ -273,6 +273,7 @@
     <string name="no_transcript_label">No transcript</string>
     <string name="remove_from_favorite_label">Remove from favorites</string>
     <string name="visit_website_label">Visit website</string>
+    <string name="visit_social_interest_label">Social comments</string>
```

Comment:
The string name has "visit" but the content hasn't. Here it says "social comments". So should the code everywhere maybe also be called "social comments" instead of "social interact" for consistency?

Reply:
Let me know if it's okay in our code, we use Social Comments, in the PodcastIndex namespace it is socialInteract.